style.css

```diff
@@ -136,3 +137,244 @@ h2 {
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.cardworks-two {
+  box-sizing: border-box;
+
+  /* Auto layout  */
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 596px;
+
+  /* Neutrals /N0 */
+  background: #ffff;
+
+  /* Neutrals /N40 */
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.cardworks-three {
+  box-sizing: border-box;
+
+  /* Auto layout  */
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+
+  /* Neutrals /N0 */
+  background: #ffff;
+
+  /* Neutrals /N40 */
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.cardworks-four {
+  box-sizing: border-box;
+
+  /* Auto layout  */
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 596px;
+
+  /* Neutrals /N0 */
+  background: #ffff;
+
+  /* Neutrals /N40 */
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.frame {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  height: 32px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.tag {
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 214px;
+  height: 24px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.buttons {
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  position: absolute;
+  width: 126px;
+  height: 48px;
+  background-color: #fff;
+}
+
+.press {
+  width: 102px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 5px;
+  border: 1px solid #396df2;
+  margin-top: 20px;
+  margin-bottom: 30px;
+  margin-left: 20px;
+
+  /* Identical to box-height or 141% */
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #396df2;
+
+  /* Inside auto layout */
+  flex: 0;
+  order: 0;
+  flex-grow: 0;
+}
+
+.about-me {
+  position: absolute;
+  width: 375px;
+  height: 1263px;
+  background: #fff;
+}
+
+.about {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.start {
+  width: 138px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 5px;
+  border: 1px solid #396df2;
+  margin-top: 20px;
+  margin-bottom: 30px;
+  margin-left: 20px;
+  background-color: #fff;
+
+  /* Identical to box-height or 141% */
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #396df2;
+
+  /* Inside auto layout */
+  flex: 0;
+  order: 0;
+  flex-grow: 0;
+}
+
```

Comment:
- [ ] Similarly, as suggested by the reviewer [here](https://github.com/brendalutiku/Portfolio/pull/2#discussion_r1154248069) please make sure to stick to the Figma design guidelines while styling button in terms of background color, paddings and width: as well 🎯 As an example, you can try something like below 👇 See the images below for more clarity 

```
    padding: 12px;
    width: 162px;
    height: 48px;
    background: transparent;
    border-radius: 8px;
    margin-left: 0;
    justify-content: center;
```

You design before 🔽 | Your design now 🔽 | Figma design 🔽
---| ---| ---|
![image](https://user-images.githubusercontent.com/106119105/229162730-a684287c-074d-4f23-b132-b64ae8d91d03.png) | ![image](https://user-images.githubusercontent.com/106119105/229162895-97f76b22-c8f4-4b0b-866c-8b2159ffd823.png) | ![image](https://user-images.githubusercontent.com/106119105/229162984-724d66a4-9627-4482-ad44-9379aac34389.png)